### PR TITLE
Oanda Sub/Unsub request grouping

### PR DESCRIPTION
**Pull request by StefanoRaggi**

### CODE REVIEW

File: Brokerages/Oanda/OandaBrokerage.DataQueueHandler.cs [hunk 1]
```diff
@@ -84,27 +94,74 @@ public void Subscribe(LiveNodePacket job, IEnumerable<Symbol> symbols)
...
+                        // restart streaming session
+                        SubscribeSymbols(symbolsToSubscribe);
+
+                        lock (_lockerSubscriptions)
+                        {
+                            _lastSubscribeRequestUtcTime = DateTime.UtcNow;
+                            if (_subscribedSymbols.Count == symbolsToSubscribe.Count)
+                            {
+                                // no more subscriptions pending, task finished
+                                _subscriptionsPending = false;
```
Comment (mchandschuh) [inline Brokerages/Oanda/OandaBrokerage.DataQueueHandler.cs]: may want to mark this field as volatile since we're on another thread

Comment (StefanoRaggi) [inline Brokerages/Oanda/OandaBrokerage.DataQueueHandler.cs]: I believe access to this bool is already synchronized with lock()

Comment (mchandschuh) [inline Brokerages/Oanda/OandaBrokerage.DataQueueHandler.cs]: Touché good sir! Looks good, thanks!

Comment (StefanoRaggi) [inline Brokerages/Oanda/OandaBrokerage.DataQueueHandler.cs]: Merged in https://github.com/QuantConnect/Lean/commit/3b8e0c64f0fb9be3c5eac2cabccec7ffc020296d

**Pull request closed without merging**